content/en/blog/2021/revision-tags/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Istio control plane canary with revisions and tags"
+description: Learn how to take more control over your Istio control-plane upgrades.
+publishdate: 2021-05-20
+attribution: "Christian Posta (Solo.io), Lin Sun (Solo.io), Sam Naser (Google)"
+keywords: [upgrades,revisions,operations,canary]
+---
+
+The Istio community [releases new versions of Istio](/about/supported-releases/) every quarter and sometimes between the normal release cadence for bug fixes [or CVE patches](/about/security-vulnerabilities/). The operator of an Istio service mesh needs to upgrade the control plane and data plane components during the life of the service mesh. You must take care when upgrading a service mesh as a mistake could take down the entire mesh and affect your business traffic. Istio has a couple mechanisms to make it safe to perform upgrades in a controlled manner, and in Istio 1.10 we further improve this operational experience.
```

Comment:
```suggestion
Like all security software, your service mesh should be kept up-to-date. The Istio community [releases new versions every quarter](/docs/releases/supported-releases/), with regular patch releases for bug fixes [and security vulnerabilities](/docs/releases/security-vulnerabilities/). The operator of a service mesh will need to upgrade the control plane and data plane components many times. You must take care when upgrading, as a mistake could affect your business traffic. Istio has many mechanisms to make it safe to perform upgrades in a controlled manner, and in Istio 1.10 we further improve this operational experience.
```

Reply:
Changes here should fix the lint issue with links.